server/sonar-server/src/main/java/org/sonar/server/ce/ws/SubmitAction.java

```diff
@@ -27,20 +27,24 @@
 import org.sonar.api.server.ws.WebService;
 import org.sonar.ce.queue.CeTask;
 import org.sonar.server.computation.queue.ReportSubmitter;
+import org.sonar.server.organization.DefaultOrganizationProvider;
 import org.sonar.server.ws.WsUtils;
 import org.sonarqube.ws.WsCe;
 
 public class SubmitAction implements CeWsAction {
 
-  public static final String PARAM_PROJECT_KEY = "projectKey";
-  public static final String PARAM_PROJECT_BRANCH = "projectBranch";
-  public static final String PARAM_PROJECT_NAME = "projectName";
-  public static final String PARAM_REPORT_DATA = "report";
+  private static final String PARAM_ORGANIZATION_KEY = "organization";
```

Comment:
This has nothing to do with scanner. Other parameters of this WS are "projectKey", "projectBranch", "projectName", ... But this is minor :)